src/styles/utils/_vars.scss

```diff
@@ -1 +1,16 @@
 $c-gray: #eee;
+$c-menu-item: #f9f9f9;
+$c-white: #fff;
+$main-text-c: #000;
+$second-text-c: #7c7c7c;
+$btn-c: #333;
+$btn-hover: #131313;
+$page-background-c: #f7f7f7;
+$see-more-link-c:#0db2b3;
+$contarast-c: #d6ecec;
+$dark-green-c: #0c797a;
+$form-border-c:#828282;
+$placeholder-c: #bdbdbd;
+$disabled-btn-c: #d0d0d0;
+$submit-btn-c: #39bebf;
```

Comment:
Do not abbreviate the names of variables to unreadable ones and try to minimally use naming according to the content (that is, if you want to keep the color white, do not make the variable white-color, but name it more universally)